### PR TITLE
Update tgi-messages-api.md

### PR DESCRIPTION
**Pull request by philschmid**

### CODE REVIEW

File: tgi-messages-api.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ for message in chat_completion:
 Behind the scenes, TGI’s Messages API automatically converts the list of messages into the model’s required instruction format using its [chat template](https://huggingface.co/docs/transformers/chat_templating). 
 
 
-> ##### 💡 Certain OpenAI features, like function calling, are not compatible with TGI. Currently, the Messages API supports the following chat completion parameters: `stream`, `max_new_tokens`, `frequency_penalty`, `logprobs`, `seed`, `temperature`, and `top_p`.
+> ##### 💡 Certain OpenAI features, like function calling, are not compatible with TGI. Currently, the Messages API supports the following chat completion parameters: `stream`, `max_tokens`, `frequency_penalty`, `logprobs`, `seed`, `temperature`, and `top_p`.
```
Comment (andrewrreed) [inline tgi-messages-api.md:124]: Are you sure? 

https://github.com/huggingface/text-generation-inference/blob/39af000cb9bfa12628c21c6e65d3ce8930250171/router/src/server.rs#L606

Comment (philschmid) [reply tgi-messages-api.md:124]: coming from here https://huggingface.co/docs/text-generation-inference/messages_api

Comment (philschmid) [reply tgi-messages-api.md:124]: Also our snippets use `max_tokens`

Comment (philschmid) [reply tgi-messages-api.md:124]: see: https://github.com/huggingface/text-generation-inference/blob/39af000cb9bfa12628c21c6e65d3ce8930250171/router/src/lib.rs#L359

Comment (philschmid) [reply tgi-messages-api.md:124]: its mapped

**Review by pcuenca — APPROVED** [review]

File: tgi-messages-api.md (hunk 1 above)
Comment (pcuenca) [reply tgi-messages-api.md:124]: There's a still an indentation issue in line 113 above, can you add a fix to this pr? Otherwise we can open another one.

**Pull request merged**